servers/rendering/renderer_rd/shaders/luminance_reduce_raster.glsl

```diff
@@ -0,0 +1,65 @@
+/* clang-format off */
+#[vertex]
+
+#version 450
+
+#VERSION_DEFINES
+
+#include "luminance_reduce_raster_inc.glsl"
+
+layout(location = 0) out vec2 uv_interp;
+/* clang-format on */
+
+void main() {
+	vec2 base_arr[4] = vec2[](vec2(0.0, 0.0), vec2(0.0, 1.0), vec2(1.0, 1.0), vec2(1.0, 0.0));
+	uv_interp = base_arr[gl_VertexIndex];
+
+	gl_Position = vec4(uv_interp * 2.0 - 1.0, 0.0, 1.0);
+}
+
+/* clang-format off */
+#[fragment]
+
+#version 450
+
+#VERSION_DEFINES
+
+#include "luminance_reduce_raster_inc.glsl"
+
+layout(location = 0) in vec2 uv_interp;
+/* clang-format on */
+
+layout(set = 0, binding = 0) uniform sampler2D source_exposure;
+
+#ifdef FINAL_PASS
+layout(set = 1, binding = 0) uniform sampler2D prev_exposure;
+#endif
+
+layout(location = 0) out highp float exposure;
```

Comment:
Calling this exposure is kinda misleading, since the division doesn't happen until the tonemapper shader file.
luminance or clamped_luminance is a better name imo.

Reply:
that is what it is called in the other shaders, both in Godot 3 and in the compute version of luminance.

That said, I fully agree with you its a better name (I would just stick with luminance) but it then makes sense to rename in the compute shader as well. @reduz your thoughts?